pkgs/servers/reproxy/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "reproxy";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "umputun";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-xZVMwa/44/hGmZxMbHGNInMjsO/5ZXalcHZq5xgO1No=";
+  };
+
+  postPatch = ''
+    # Requires network access
+    substituteInPlace app/main_test.go \
+      --replace "Test_Main" "Skip_Main"
+  '';
+
+  vendorSha256 = null;
+
+  postInstall = ''
+    mv $out/bin/{app,reproxy}
+  '';
```

Comment:
According to the [`Makefile`](https://github.com/umputun/reproxy/blob/master/Makefile#L21) it looks like that by using `buildFlagsArray` the output could be defined. Thus, not renaming needed.

With `main.revision=$(version)` in the `buildFlagsArray` probably the version could show up in the output.

```bash
ew/pr-123037/results/reproxy/bin]$ ./reproxy -h
reproxy unknown
[...]
```

Reply:
> According to the [`Makefile`](https://github.com/umputun/reproxy/blob/master/Makefile#L21) it looks like that by using `buildFlagsArray` the output could be defined. Thus, not renaming needed.
>

Well, I've replaced with explicit `installPhase`. I don't see how to do it properly with `buildFlagsArray`.
 
> With `main.revision=$(version)` in the `buildFlagsArray` probably the version could show up in the output.
> 
> ```shell
> ew/pr-123037/results/reproxy/bin]$ ./reproxy -h
> reproxy unknown
> [...]
> ```

Fixed.